README.md

```diff
@@ -1,25 +1,225 @@
+
+> ⛽️ You'll need to get some funds from the faucet for gas.
+
+![gas&wallet](https://github.com/scaffold-eth/se-2-challenges/assets/80153681/912d0d4b-db34-49d3-bd7d-7ca0ab18eb66)
+
+> 🦊 At first, **don't** connect MetaMask. If you are already connected, click **Disconnect**:
+
+<p>
+  <img src="https://github.com/scaffold-eth/se-2-challenges/assets/80153681/2c7a1e40-50ad-4c20-ba3e-a56eff4b892b" width="33%" />
+  <img src="https://github.com/scaffold-eth/se-2-challenges/assets/80153681/1bcf9752-e8ae-4db6-a0a6-5dc774abe46c" width="33%" />
+</p>
+
+> 🔥 We'll use burner wallets on localhost.
+
+> 👛 Explore how burner wallets work in 🏗 Scaffold-ETH 2 by opening a new incognito window and navigate to http://localhost:3000. You'll notice it has a new wallet address in the top right. Copy the incognito browser's address and send localhost test funds to it from your first browser (using the **Faucet** button in the bottom left):
+
+![icognito&webBrowser](https://github.com/scaffold-eth/se-2-challenges/assets/80153681/fd191447-a31f-4c03-a36f-936bfb70c2a1)
+
+> 👨🏻‍🚒 When you close the incognito window, the account is gone forever. Burner wallets are great for local development but you'll move to more permanent wallets when you interact with public networks.
+
+---
+
+## Checkpoint 2: 🖨 Minting
+
+> ✏️ Mint some NFTs! Click the **MINT NFT** button in the `My NFTs` tab.
+
+![image](https://github.com/scaffold-eth/se-2-challenges/assets/80153681/74cf02f2-4c1b-4278-9841-f19f668e0b1e)
+
+👀 You should see your NFTs start to show up:
+
+![image](https://github.com/scaffold-eth/se-2-challenges/assets/80153681/63dabceb-ad42-4c09-8e5d-a0139939e32d)
+
+👛 Open an incognito window and navigate to http://localhost:3000
+
+🎟 Transfer an NFT to the incognito window address using the UI:
+
+![image](https://github.com/scaffold-eth/se-2-challenges/assets/80153681/3b92fb50-d43f-48a8-838c-c45c443b0b71)
+
+👛 Try to mint an NFT from the incognito window.
+
+> Can you mint an NFT with no funds in this address? You might need to grab funds from the faucet to pay for the gas!
+
+🕵🏻‍♂️ Inspect the `Debug Contracts` tab to figure out what address is the owner of YourCollectible?
+
+🔏 You can also check out your smart contract `YourCollectible.sol` in `packages/hardhat/contracts`.
+
+💼 Take a quick look at your deploy script `00_deploy_your_contract.js` in `packages/hardhat/deploy`.
+
+📝 If you want to edit the frontend, navigate to `packages/nextjs/app` and open the specific page you want to modify. For instance: `/myNFTs/page.tsx`. For guidance on [routing](https://nextjs.org/docs/app/building-your-application/routing/defining-routes) and configuring [pages/layouts](https://nextjs.org/docs/app/building-your-application/routing/pages-and-layouts) checkout the Next.js documentation.
+
+---
+
+## Checkpoint 3: 💾 Deploy your contract! 🛰
+
+🛰 Ready to deploy to a public testnet?!?
+
+> Change the defaultNetwork in `packages/hardhat/hardhat.config.ts` to `sepolia`.
```

Comment:
The new option to deploy to Optimism Sepolia is missing in this section. https://speedrunethereum.com/challenge/simple-nft-example